addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechPanel.java

```diff
@@ -174,7 +175,24 @@ private JScrollPane getJScrollPane() {
 
     protected JXTable getTechTable() {
         if (techTable == null) {
-            techTable = new JXTable(techModel);
+            techTable =
+                    new JXTable(techModel) {
+                        private static final long serialVersionUID = -5249686560976842645L;
+
+                        @Override
+                        public String getToolTipText(MouseEvent e) {
+                            String tip = "";
+                            int rowIndex = rowAtPoint(e.getPoint());
+
+                            try {
+                                tip = techModel.getApp(rowIndex).getDescription();
+                            } catch (RuntimeException e1) {
+                                // catch null pointer exception if mouse is over an empty line
+                            }
+
+                            return tip.isEmpty() ? null : tip;
```

Comment:
If empty string is returned then a weird blank box is shown (in dark mode anyway).